app/services/nutritionix/api_client.rb

```diff
@@ -0,0 +1,105 @@
+require 'httparty'
+
+module Nutritionix
+
+# ---------------------------can't get calories
+# class ApiClient
+#   include HTTParty
+#   format :json
+#   base_uri "https://api.nutritionix.com"
+#
+#   def initialize()
+#   end
+#
+#   def get(product)
+#     api_key = Rails.application.credentials.dig(:nutritionix, :api_key)
+#     app_id = Rails.application.credentials.dig(:nutritionix, :app_id)
+#     params = {
+#               headers: { "Content-Type":"application/json" },
+#               body: [{ appId: app_id,
```

Comment:
I'm not sure if it's necessary to convert everything to json and send as body, did you try sending parameters with `query` option? (similar to my comment below)